tools/install_os_deps.sh

```diff
@@ -6,10 +6,11 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
     if [[ ! -x $BREW ]]; then
         /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
     fi
+    # Update PATH to prefer Homebrew's bison over the macOS-provided version
+    export PATH="/usr/local/opt/bison/bin:$PATH"
```

Comment:
This is unlikely to work as Travis will spawn different shells. You need to do what brew recommends:
`export PATH="/usr/local/opt/bison/bin:$PATH"' >> ~/.bash_profile`
However, you also need the links, because by default bison will be installed in `/usr/local/Cellar/bison/<version>/`.

And mind, you, I have 10.13.6, and here is the result of my linking:
```
brew unlink bison && brew link --verbose --force bison
Unlinking /usr/local/Cellar/bison/3.0.4_1... 9 symlinks removed
Linking /usr/local/Cellar/bison/3.0.4_1...
ln -s ../Cellar/bison/3.0.4_1/bin/bison bison
ln -s ../Cellar/bison/3.0.4_1/bin/yacc yacc
ln -s ../../Cellar/bison/3.0.4_1/share/aclocal/bison-i18n.m4 bison-i18n.m4
ln -s ../Cellar/bison/3.0.4_1/share/bison bison
ln -s ../../Cellar/bison/3.0.4_1/share/doc/bison bison
ln -s ../../Cellar/bison/3.0.4_1/share/info/bison.info bison.info
info /usr/local/share/info/bison.info
ln -s ../../../Cellar/bison/3.0.4_1/share/man/man1/bison.1 bison.1
ln -s ../../../Cellar/bison/3.0.4_1/share/man/man1/yacc.1 yacc.1
ln -s ../Cellar/bison/3.0.4_1/lib/liby.a liby.a
9 symlinks created

If you need to have this software first in your PATH instead consider running:
  echo 'export PATH="/usr/local/opt/bison/bin:$PATH"' >> ~/.bash_profile
brew --version
Homebrew 1.7.3-88-g45bbf04
Homebrew/homebrew-core (git revision 1ad2f; last commit 2018-09-09)
```

Reply:
Btw, this is being actively discussed on the Homebrew repo: 
https://github.com/vgteam/vg/issues/1915